src/instructlab/eval/mt_bench_common.py

```diff
@@ -247,37 +251,99 @@ def play_a_match_single(
     return result
 
 
+def _is_fatal_openai_error(e: openai.OpenAIError) -> bool:
+    return isinstance(
+        e,
+        (
+            openai.APIConnectionError,
+            openai.AuthenticationError,
+            openai.PermissionDeniedError,
+            openai.NotFoundError,
+        ),
+    )
+
+
+# TODO: copied from instructlab (cli) utils module; consolidate somewhere?
```

Comment:
can we just import this from the cli? if we've made sure to keep our imports tight everywhere we may not have to worry about import cycles in that case

Reply:
oh yeah, you're right